src/cli/restore/restore.go

```diff
@@ -82,14 +82,17 @@ func runRestore(
 		return Only(ctx, clues.Wrap(err, "Failed to run "+serviceName+" restore"))
 	}
 
-	Info(ctx, "Completed Restore:")
+	Info(ctx, "Restore Complete")
```

Comment:
Is this going to break sanity tests and other similar github workflows? I know some of those check for specific output strings

Reply:
Hopefully not!  I'll go check on that.